package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "argo-clinical",
-  "version": "1.48.1",
+  "version": "1.48.2",
```

Comment:
please use minor versions `1.49.0`  and leave the patch for hot fixes i(i.e. branched of master) if needed